core/azure-core-amqp/pom.xml

```diff
@@ -7,13 +7,13 @@
   <parent>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-parent</artifactId>
-    <version>1.0.0-preview.1</version>
+    <version>1.1.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
   <groupId>com.azure</groupId>
   <artifactId>azure-core-amqp</artifactId>
-  <version>1.0.0-preview.1</version>
```

Comment:
Should be changed to `1.0.0-preview.3`. The diff here looks a little odd as it is `1.0.0-preview.2` in master (https://github.com/Azure/azure-sdk-for-java/blob/master/core/azure-core-amqp/pom.xml#L16), you might want to rebase your changes on the latest master as I think some of these were bumped as part of a respin for the first preview.